app/screens/map/MapContainer.spec.js

```diff
@@ -80,4 +80,30 @@ describe('screens/map/MapContainer', () => {
       ]);
     });
   });
+
+  describe('componentDidUpdate', () => {
```

Comment:
It shouldn't be possible for the boundaries to have changed with isLoaded being false, so I didn't add a test for that. But I added a test to make sure onMapRef doesn't call the function if isLoaded is false, as that is the purpose of having isLoaded in the code.